### PR TITLE
[SPARK-11552][DOCS][Replaced example code in ml-decision-tree.md using include_example]

### PR DESCRIPTION
**Pull request by agsachin**
I have tested it on my local, it is working fine, please review

### CODE REVIEW

File: examples/src/main/java/org/apache/spark/examples/ml/JavaDecisionTreeClassificationExample.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// scalastyle:off println
+package org.apache.spark.examples.ml;
+
+// $example on$
+
```
Comment (yinxusen) [inline examples/src/main/java/org/apache/spark/examples/ml/JavaDecisionTreeClassificationExample.java]: remove this empty line

File: examples/src/main/java/org/apache/spark/examples/ml/JavaDecisionTreeRegressionExample.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// scalastyle:off println
+package org.apache.spark.examples.ml;
+// $example on$
```
Comment (yinxusen) [inline examples/src/main/java/org/apache/spark/examples/ml/JavaDecisionTreeRegressionExample.java]: remove empty line here

File: examples/src/main/java/org/apache/spark/examples/ml/JavaDecisionTreeRegressionExample.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import org.apache.spark.ml.feature.VectorIndexer;
+import org.apache.spark.ml.feature.VectorIndexerModel;
+import org.apache.spark.ml.regression.DecisionTreeRegressionModel;
+import org.apache.spark.ml.regression.DecisionTreeRegressor;
+import org.apache.spark.mllib.regression.LabeledPoint;
+import org.apache.spark.mllib.util.MLUtils;
+import org.apache.spark.rdd.RDD;
+import org.apache.spark.sql.DataFrame;
+import org.apache.spark.sql.SQLContext;
+
```
Comment (yinxusen) [inline examples/src/main/java/org/apache/spark/examples/ml/JavaDecisionTreeRegressionExample.java]: ditto

**Comment by agsachin** [comment]
Thanks @yinxusen
I have removed all the empty lines

**Comment by yinxusen** [comment]
@mengxr LGTM

**Comment by mengxr** [comment]
ok to test

**Comment by mengxr** [comment]
test this please

**Comment by mengxr** [comment]
Merged into master and branch-1.6. Thanks!

**Pull request closed without merging**